src/OCC/utils/utils.cpp

```diff
@@ -144,4 +144,17 @@ std::vector<uint32_t> getPath(char *pathStr)
         token = strtok(NULL, "/");
     }
     return myPath;
+}
+
+/**
+ * Sends an error message using OSC protocol.
+ *
+ * @param address The address to send the error message to.
```

Comment:
There is no address in the method. old method docs? or something missing? where is the message sent to?

Reply:
thanks i missed that one. this part of the doc is removed now